.travis.yml

```diff
@@ -1,16 +1,27 @@
 language: go
 
+# cache:
+#   directories:
+#   - $HOME/.glide
+
 go:
 - 1.7.1
 
 branches:
   only:
   - master
   - cluster-up
+  - /^issue-[0-9]+$/
```

Comment:
This is probably we need to decide whether we use `issue-*` pattern or `fix-*` pattern. 
cc @LalatenduMohanty